bioblend/galaxyclient.py

```diff
@@ -34,6 +34,7 @@ def __init__(
         self,
         url: str,
         key: Optional[str] = None,
+        token: Optional[str] = None,
```

Comment:
Tests are failing because you need to add this parameter also to `bioblend/galaxy/__init__.py:GalaxyInstance.__init__()`, `bioblend/galaxy/objects/galaxy_instance.py:GalaxyInstance.__init__()` and `bioblend/toolshed/__init__.py:ToolShedInstance.__init__()` methods, which were calling this one but passing email to token, password to email and so forth.
Also, can you rebase on the latest `main`? I've just pushed commit 14d9e029c7cfe58f4b46114f4e6a5510fdba23b4 which change these methods to pass the parameters by keyword.
I'd also move `token` after `password` (actually after `*` in the rebase) to not break the order of the positional parameters.